packages/z2s/src/compiler.ts

```diff
@@ -397,4 +398,14 @@ export class Compiler {
     const mapped = this.#nameMapper.tableName(table);
     return sql.ident(mapped);
   }
+
+  #selectCol(table: string, column: string, type: ValueType) {
```

Comment:
Might need bigint handling, bigint-json. And we'll want to double-check `numeric`. We represent it as a number but PG might convert it to a string when serializing to JSON.

Reply:
bigint and numeric get converted as we need

<img width="689" alt="image" src="https://github.com/user-attachments/assets/3f95c49e-d553-481d-8381-6ec41728a42b" />

<img width="697" alt="image" src="https://github.com/user-attachments/assets/6e847242-fc70-4e37-ab93-cbcea91e87ec" />